tests/unit/agents/AgentAdapters.test.ts

```diff
@@ -24,7 +24,7 @@ describe('Agent Adapters', () => {
   });
 
   describe('CopilotAgent', () => {
-  it('backs up and writes copilot-instructions.md', async () => {
+    it('backs up and writes copilot-instructions.md', async () => {
```

Comment:
Sorry there's a large diff for this one, I tidied up the indenting and moved the tests that were outside the describe into their respective scope.